code/datums/customization_styles.dm

```diff
@@ -737,9 +737,9 @@ TYPEINFO(/datum/customization_style/hair/gimmick)
 				name = "Croft"
 				id = "croft"
 				gender = FEMININE
-			indian
+			doublebraids
 				name = "Double Braids"
-				id = "indian"
+				id = "doublebraids"
```

Comment:
This will break existing profiles that use this hairstyle, ids are meant to be immutable because they're used in stored savefiles. If you want to change it you'll need to include a conversion snippet somewhere in appearance loading code.

Reply:
Miiiight be better to just do this alongside a big save conversion that's gonna be happening as part of #21026? I think having lots and lots of conversion snippets is not good, and if we're already planning on one, might as well lump it together. Pinging @Tarmunora for input.

Reply:
> This will break existing profiles that use this hairstyle, ids are meant to be immutable because they're used in stored savefiles. If you want to change it you'll need to include a conversion snippet somewhere in appearance loading code.

Yup, good call. I'll let it get changed in a future fix!